source/components/recipe-contribute/recipe-contribute.css

```
@@ -76,3 +77,19 @@ textarea {
 .red-star {
   color: red;
 }
+
+.category-checkbox {
+  cursor: pointer;
+  height: 25px;
+  width: 25px;
+  margin-top: 30px;
```

Let's change this to margin bottom so there's less spacing between the "Categories" header and the first checkbox. Also I think we should add a little margin right so the checkbox is spaced out more from the label. See below:
```suggestion
  margin-bottom: 30px;
  margin-right: 10px;
```